metashare/api/jobs.py

```diff
@@ -432,3 +432,20 @@ def populate_github_users(repository):
 
 
 populate_github_users_job = job(populate_github_users)
+
+
+def submit_review(*, user, scratch_org, data):
+    try:
+        repository = get_repo_info(
+            user, repo_id=scratch_org.task.project.repository.repo_id
+        )
+        pr = repository.pull_request(scratch_org.task.pr_nuber)
+        pr.create_review(data["notes"], event=data["status"])
```

Comment:
@wlonk It's not clear to me from our earlier discussion whether we actually want to be creating a GH "review". Our earlier conversation was about just submitting it as a comment on the PR (see https://trello.com/c/BegMDQLo/58-facilitate-qa-results-submission-through-metashare-and-github-commit-status#comment-5e31a28af23bb74c4e2fdce1).

Regardless, I also think we need a MetaShare commit status check on GitHub?

Reply:
The comment in the link about "status of a review (approved or changes-requested)" implied to me that it would be a GitHub review object; even though we could make those statuses locally, I don't understand the merit of only submitting comments but marking them local-to-MetaShare as "approved" or "changes-requested".

Reply:
Ok -- I think that's up to @davisagli and Jason. The use-case I could see would be if they _don't_ want MetaShare to impact the PR review status. MS review in this case is functional review of the org, not code review. The PR review status implies code review, and also could impact ability-to-merge depending on GH protection settings -- I'm not 100% sure that's what we want in this case.

Reply:
Mmm, that distinction makes sense to me, gotcha.

Reply:
Confirmed that with @davisagli and we're good to go with the current "always a comment" appraoch.